src/js/common/utils/validations.js

```diff
@@ -129,6 +131,26 @@ function isValidPartialDate(day, month, year) {
   return true;
 }
 
+function isValidMonthYear(month, year) {
+  if (typeof month === 'object') {
```

Comment:
I kept passing a date field into this function (instead of a month and year) and having confusing validation output as a result. We don't have to have this, but seemed useful.